src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -128,6 +129,36 @@ public void testKeyEncryption() throws InvalidParametersException {
     }
 
     @Test
+    public void testDefaultAesCipherParams() throws Exception {
+        //plaintext is less than 1 block long
+        String plaintextLength28 = StringHelper.generateStringOfLength(7, 'A');
+        assertEncryptionUsesExpectedDefaultParams(plaintextLength28);
```

Comment:
@Neurrone Is 28 a deliberate choice? I was expecting to see a boundary value test case here. i.e. the longest value this is still less than a block.